src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -1040,25 +1097,63 @@ end finite_measure --namespace
 
 end normalize_finite_measure -- section
 
+section conditioned_probability_measure
+
+namespace probability_measure
+
+variables {Ω : Type*} [measurable_space Ω]
+
+/-- Probability measure P conditioned on an event A. -/
+def conditioned (P : probability_measure Ω) (A : set Ω) : probability_measure Ω :=
+@finite_measure.normalize Ω (nonempty_of_probability_measure P) _ (P.to_finite_measure.restrict A)
```

Comment:
Are you aware of `[probability_theory.cond](https://leanprover-community.github.io/mathlib_docs/probability/conditional_probability.html#probability_theory.cond)`. I think it would be better to define this in terms of `probability_theory.cond` so you have access to the APIs more directly